kubectl-rook-ceph.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eEuo pipefail
+
+usages() {
+  cat <<EOF
+Usage: kubectl rook [commands] [args] [<flags>]
+Provide common management and troubleshooting tools for the Rook Ceph storage provider as a Krew plugin.
+
+commands:
+  ceph
+args:
+  any ceph argument example: status
+flags:
+  -f  --format   format output in json, json-pretty etc.
```

Comment:
I read this as documentation for the `kubectl rook-ceph` command and not for the `ceph` subcommand. We probably don't need to do a multi-command help tree, but we can clean this up I think.

```
commands:
  ceph
    args  - any ceph argument (e.g., status)
    flags - any ceph flag (e.g., --format=json-pretty)
```